To: vim_dev@googlegroups.com
Subject: Patch 7.3.342
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.342
Problem:    Code not in Vim style.
Solution:   Fix the style. (Elias Diem)
Files:	    src/os_amiga.c, src/os_mac_conv.c, src/os_win16.c


*** ../vim-7.3.341/src/os_amiga.c	2010-08-15 21:57:27.000000000 +0200
--- src/os_amiga.c	2011-10-20 18:19:45.000000000 +0200
***************
*** 1022,1028 ****
  
      /* insure longword alignment */
  #ifdef __amigaos4__
!     if(!(id = AllocDosObject(DOS_INFODATA, 0)))
  	goto out;
  #else
      id = (struct InfoData *)(((long)id_a + 3L) & ~3L);
--- 1022,1028 ----
  
      /* insure longword alignment */
  #ifdef __amigaos4__
!     if (!(id = AllocDosObject(DOS_INFODATA, 0)))
  	goto out;
  #else
      id = (struct InfoData *)(((long)id_a + 3L) & ~3L);
*** ../vim-7.3.341/src/os_mac_conv.c	2011-06-13 02:03:55.000000000 +0200
--- src/os_mac_conv.c	2011-10-20 18:19:45.000000000 +0200
***************
*** 77,83 ****
  	*unconvlenp = 0;
      cfstr = CFStringCreateWithBytes(NULL, ptr, len, from, 0);
  
!     if(cfstr == NULL)
  	fprintf(stderr, "Encoding failed\n");
      /* When conversion failed, try excluding bytes from the end, helps when
       * there is an incomplete byte sequence.  Only do up to 6 bytes to avoid
--- 77,83 ----
  	*unconvlenp = 0;
      cfstr = CFStringCreateWithBytes(NULL, ptr, len, from, 0);
  
!     if (cfstr == NULL)
  	fprintf(stderr, "Encoding failed\n");
      /* When conversion failed, try excluding bytes from the end, helps when
       * there is an incomplete byte sequence.  Only do up to 6 bytes to avoid
*** ../vim-7.3.341/src/os_win16.c	2010-12-17 20:23:56.000000000 +0100
--- src/os_win16.c	2011-10-20 18:19:45.000000000 +0200
***************
*** 243,251 ****
  	/* Wait for the command to terminate before continuing */
  	while (GetModuleUsage((HINSTANCE)h_module) > 0 && again )
  	{
! 	    while( PeekMessage( &msg, NULL, 0, 0, PM_REMOVE ) && again )
  	    {
! 		if(msg.message == WM_QUIT)
  
  		{
  		    PostQuitMessage(msg.wParam);
--- 243,251 ----
  	/* Wait for the command to terminate before continuing */
  	while (GetModuleUsage((HINSTANCE)h_module) > 0 && again )
  	{
! 	    while (PeekMessage(&msg, NULL, 0, 0, PM_REMOVE) && again)
  	    {
! 		if (msg.message == WM_QUIT)
  
  		{
  		    PostQuitMessage(msg.wParam);
*** ../vim-7.3.341/src/version.c	2011-10-20 18:17:38.000000000 +0200
--- src/version.c	2011-10-20 18:21:43.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     342,
  /**/

-- 
You are only young once, but you can stay immature indefinitely.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
